### PR TITLE
[Continuous batching] Enable the rest of python tests

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Review by mzegla — APPROVED** [review]

File: tests/python_tests/test_sampling.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def test_post_oom_health(tmp_path, sampling_config):
     # Low cache size to trigger OOM quickly
     scheduler_config.num_kv_blocks = 10
     generation_configs = [generation_config]
-    model_id : str = "facebook/opt-125m"
+    model_id : str = "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
```
[the remark this answers is not in the document]
Comment (Wovchena) [reply tests/python_tests/test_sampling.py:306]: I'm still consulting on that, but opt-125m doesn't look friendly

[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply tests/python_tests/test_sampling.py:306]: why? it's several times smaller than Tiny LLama

[the remark this answers is not in the document]
Comment (Wovchena) [reply tests/python_tests/test_sampling.py:306]: I missed the word license

[the remark this answers is not in the document]
Comment (mzegla) [reply tests/python_tests/test_sampling.py]: What is wrong with the opt-125m license? Were there some changes there recently?

[the remark this answers is not in the document]
Comment (Wovchena) [reply tests/python_tests/test_sampling.py]: > You will not ... copy ...  for (i) any commercial or production purposes

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**